bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/ndk/NativeBridge.java

```diff
@@ -256,11 +261,29 @@ private boolean is32bit() {
         return is32bit;
     }
 
+    // TODO temporary - will be removed in later commit
```

Comment:
Is this change made necessary by the refactor? If so, what later commit will remove it and can you add some discussion on the PR about it?

If it's not related to the refactor, I suggest we remove this from the refactor entirely and address as a separate change, targeting master.

Reply:
Yes - the spec states that `breadcrumb.timestamp` should be a `Date`, whereas before it was a `String`. `NativeBridge` does not have access to `DateUtils` because it is defined in a separate package, meaning it is necessary to perform the conversion elsewhere.

This will be resolved by the following commit, which alters the `ADD_BREADCRUMB` message to use individual primitive fields rather than the `Breadcrumb` object - meaning that the date serialization is done up front when sending the message: https://github.com/bugsnag/bugsnag-android/commit/a581c825eeb4f510ebd2295914246f7ba096eaa2#diff-a28efd499ea99991ba5a7fb937719768R87

Completing this change without a temporary duplication either requires waiting for #659 to be merged, creating a temporary fix that will be overwritten, or just accepting that this will break CI.

Reply:
OK - understood.